apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -6,7 +6,9 @@ import { Link } from 'react-router-dom';
 import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
 import { NavMenuContainer } from '../nav-menu/nav-menu-container';
 
-const AppBarWithTestIdDataAttribute = ({ children }) => <AppBar data-testid={'app-bar'}>{children}</AppBar>;
+const AppBarWithTestIdDataAttribute = ({ children }) => (
+    <AppBar data-testid={'app-bar'}>{children}</AppBar>
```

Comment:
Just a minor syntax thing for you and @aaron-plahn: can this also be done as `<AppBar data-testid="app-bar">` without the curly braces?